utils/js/LogUtils.ts

```diff
@@ -20,60 +21,77 @@ import {castArray, isString} from 'lodash';
  * @param fn - function to execute
  * @param source - class, function or string to label the source of the message
  */
-export function withInfo<T>(msgs: string[] | string, fn: () => T, source?: any): T {
+export function withInfo<T>(msgs: Some<any>, fn: () => T, source?: any): T {
```

Comment:
Think we should type these as Some<unknown>

Reply:
Changed signature on public APIs. Ran into type errors if I changed impl methods that I did not understand.